sklearn/pipeline.py

```diff
@@ -28,7 +28,8 @@
 
 
 class Pipeline(_BaseComposition):
-    """Pipeline of transforms with a final estimator.
+    """
```

Comment:
@scikit-learn/core-devs , This kind of change is in contradiction to https://numpydoc.readthedocs.io/en/latest/format.html and seems needless... do we really want to waste our time changing this everywhere?

Reply:
I'm with you on avoiding cosmetic changes for the sake of cosmetics.

I think that there might be a potential value is if we could have a validator in our CI. The value that I might see is avoiding to merge incorrectly-formatted docstrings.

Reply:
We found about it a bit late. We disable this checking in master and we will revert these all together.

Reply:
This is pretty frustrating for the sprint attendees (but I agree it's probably the right thing to do).

Reply:
+1 to revert those, the corresponding upstream issue is https://github.com/numpy/numpydoc/issues/241 (and to a lesser extent https://github.com/numpy/numpydoc/issues/242)